# Definition of ROS service LocSetLinesForSupportActiveSrv for sick localization
# Actives / deactivates the use of the magnetic /optical line sensor for localization.
# Example call (ROS1):
# rosservice call LocSetLinesForSupportActiveSrv "{active: 1}"
# Example call (ROS2):
# ros2 service call LocSetLinesForSupportActiveSrv sick_lidar_localization/srv/LocSetLinesForSupportActiveSrvSrv "{active: 1}"
# 

# 
# Request (input)
# 

bool active # Whether magnetic lines shall be used for localization. {0 (false), 1 (true)} 

---

# 
# Response (output)
# 

bool success # true: success response received from localization controller, false: service failed (timeout or error status from controller)
